source/includes/v2/_facets.md

```diff
@@ -1,79 +1,110 @@
 # Facets
 
-A facet can be considered as a valued tag associated with a dataset or a record. For example, datasets of a catalog could have `language` as facet: associated values for this facet could be `English`, `French` or `Swedish`.
+A facet can be considered as a valued tag associated with a dataset or a record. For example, datasets of a catalog could have `language` as a facet: associated values for this facet could be `English` or `French`.
 
-In an Opendatasoft portal, facets are especially used for building the left filtering column of both the datasets catalog and dataset records exploration pages.
+In an Opendatasoft portal, facets are specially used for building the left filtering column of both the datasets catalog and dataset records exploration pages.
 
 ## Enumerating facets values
 
 Enumerating facets values allows to discover the existing values for a facet.
 
-For example, an API call to enumerate facets in a catalog can show that there are 13 datasets in English, 10 in French and 2 in Swedish.
+For example, an API call to enumerate facets in a catalog can show that there are 12 datasets in English and 1 in French.
 
 ### Catalog
 
 ```shell
-curl 'https://examples.opendatasoft.com/api/v2/catalog/facets'
+curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/facets'
```

Comment:
I see 11 and 1 but that's probably same issue as above

Reply:
That's the permission issue indeed. I'll edit this example while being logged out to make sure the examples is exactly the response returned to non-admin users.